documentation/modules/upgrading/proc-upgrade-cluster-operator.adoc

```diff
@@ -36,6 +36,16 @@ kubectl apply -f install/cluster-operator
 ----
 +
 Wait for the rolling updates to complete.
++
+The Cluster Operator might return an error message similar to:
++
+[source,shell,subs="+quotes"]
+----
+"Version {KafkaVersionLower} is not supported. Supported versions are: #...".
+----
++
+This error occurs when upgrading to an Operator version that drops support for the previous Kafka version. 
+You will upgrade to a supported Kafka version in the next procedure, when you upgrade all brokers to a later Kafka version.
 
```

Comment:
At the moment, this is presented as "information only"? 
Is that right?
Or, does the user need to update `spec.Kafka.version` now?